README.md

```diff
@@ -19,6 +19,7 @@ spree:
     DB_HOST: postgres
     DB_PORT: 5432
 ``` 
+- (default) use the `DATABASE_URL` environment variable
```

Comment:
I think the lines above this one won't be valid anymore, right?

Reply:
In theory you can use both the DATABASE_URL envvar and database.yml. But since our docker-compose basically imposes the use of DATABASE_URL it's not worth doing I guess. You are right, I'll remove it.